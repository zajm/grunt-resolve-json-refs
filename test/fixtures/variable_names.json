{
    "SIMPLE": "Simple",
    "WITH_UNDERSCORE": "With Underscore",
    "NUMBER123": "With Number",
    "_BEGINNING_WITH_UNDERSCORE": "Beginning with underscore",
    "lowercase": "lowercase",
    "&%)": "special chars",
    "INCLUDE_SIMPLE": "@:SIMPLE",
    "INCLUDE_WITH_UNDERSCORE": "@:WITH_UNDERSCORE",
    "INCLUDE_NUMBER123": "@:NUMBER123",
    "INCLUDE_BEGINNING_WITH_UNDERSCORE": "@:_BEGINNING_WITH_UNDERSCORE",
    "INCLUDE_lowercase": "@:lowercase",
    "INCLUDE_&%)": "@:&%)"
}