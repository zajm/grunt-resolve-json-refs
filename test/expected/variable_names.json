{
    "SIMPLE": "Simple",
    "WITH_UNDERSCORE": "With Underscore",
    "NUMBER123": "With Number",
    "_BEGINNING_WITH_UNDERSCORE": "Beginning with underscore",
    "lowercase": "lowercase",
    "&%)": "special chars",
    "INCLUDE_SIMPLE": "Simple",
    "INCLUDE_WITH_UNDERSCORE": "With Underscore",
    "INCLUDE_NUMBER123": "With Number",
    "INCLUDE_BEGINNING_WITH_UNDERSCORE": "@:_BEGINNING_WITH_UNDERSCORE",
    "INCLUDE_lowercase": "@:lowercase",
    "INCLUDE_&%)": "@:&%)"
}